charts/steward/README.md

```diff
@@ -145,9 +145,11 @@ Common parameters:
 | Parameter | Description | Default |
 |---|---|---|
 | <code>pipelineRuns.<wbr/>logging.<wbr/>elasticsearch.<wbr/>indexURL</code> | (string)<br/> The URL of the Elasticsearch index to send logs to. If null or empty, logging to Elasticsearch is disabled. Example: `http://elasticsearch-master.elasticsearch.svc.cluster.local:9200/jenkins-logs/_doc` | empty |
-| <code>pipelineRuns.<wbr/>jenkinsfileRunner.<wbr/>image.<wbr/>repository</code> | (string)<br/> The container registry and repository of the Jenkinsfile Runner image. | `stewardci/stewardci-jenkinsfile-runner` |
-| <code>pipelineRuns.<wbr/>jenkinsfileRunner.<wbr/>image.<wbr/>tag</code> | (string)<br/> The tag of the Jenkinsfile Runner image in the container registry. | A fixed image tag. |
-| <code>pipelineRuns.<wbr/>jenkinsfileRunner.<wbr/>image.<wbr/>pullPolicy</code> | (string)<br/> The image pull policy for the Jenkinsfile Runner image. For possible values see field `imagePullPolicy` of the `container` spec in the Kubernetes API documentation. | `IfNotPresent` |
+| <code>pipelineRuns.<wbr/>jenkinsfileRunner.<wbr/>image.<wbr/>repository</code> | OUTDATED (string)<br/> Use <code>pipelineRuns.<wbr/>jenkinsfileRunner.<wbr/>image</code> instead. | |
+| <code>pipelineRuns.<wbr/>jenkinsfileRunner.<wbr/>image.<wbr/>tag</code> | OUTDATED (string)<br/> Use <code>pipelineRuns.<wbr/>jenkinsfileRunner.<wbr/>image</code> instead.  | |
+| <code>pipelineRuns.<wbr/>jenkinsfileRunner.<wbr/>image.<wbr/>pullPolicy</code> | OUTDATED (string)<br/> Use <code>pipelineRuns.<wbr/>jenkinsfileRunner.<wbr/>imagePullPolicy</code> instead. | |
```

Comment:
I wonder where we configure(d) pull secrets for the JFR Image. Or did we not support image pull secrets yet - and still don't do (for now)?

Reply:
This is working with the spec.imagePullSecrets (see run_types.go line 58).
See also runManager.go line 245ff. There the pull secrets are copied to the run namespace.

Reply:
This part was not changed at all.